src/backend/kvm/thread.rs

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use super::super::Command;
+
+use std::sync::{Arc, RwLock};
+
+use anyhow::{anyhow, Result};
+use kvm_ioctls::{VcpuExit, VcpuFd};
+use primordial::{Address, Register};
+use sallyport::syscall::enarx::MemInfo;
+use sallyport::syscall::{SYS_ENARX_BALLOON_MEMORY, SYS_ENARX_MEM_INFO};
+use sallyport::KVM_SYSCALL_TRIGGER_PORT;
+use sallyport::{Block, Reply};
+
+pub struct Thread {
+    keep: Arc<RwLock<super::Keep>>,
+    vcpu_fd: VcpuFd,
+}
+
+impl Drop for Thread {
+    fn drop(&mut self) {
+        //
+    }
+}
```

Comment:
What is this for?

Reply:
Shouldn't dropping the thread put the `cpu_fd` back in the list?

Reply:
yeah, you are right